src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const weekRentOff = 50;
+  const threeDaysOff = 20;
+  const threeDaysRent = 3;
+  const weekRent = 7;
```

Comment:
What if it changes to 8 days? You'll have to rename it.
```suggestion
  const minDaysForBigDiscount = 7;
```